common/util.go

```diff
@@ -44,6 +41,15 @@ func VerifyLinks(links []string) ([]string, error) {
 	}
 }
 
+func stripRefFromLink(link string) string {
+	idx := strings.Index(link, "?ref")
+	if idx != -1 {
+		return link[:idx]
+	} else {
+		return link
```

Comment:
Please move this outside of the `else` block and remove the block.

Reply:
fixed with 20c6ee9